### ISSUE TITLE
the AP of train model

### ISSUE THREAD

**Issue by MillX2021** [SPengLiang/DID-M3D#5]
HI, thanks for your work and paper.
I used your pertrained model to test the ap of kitti,it could get AP 25.38 (so well).
But I used my model of use the train code by 150 epoch, only get AP 21.79.(batch size = 16, only 1 gpu)。
why so big gap  of this? and what can i do something for get AP of near 25？

**Comment by SPengLiang** [comment]
Thanks for your feedback.  Can you post your log and the overall performance? Actually, we have not trained the model on a single GPU. We will try to reproduce your results and give you feedback as soon as possible.

**Comment by MillX2021** [comment]
log file in here
[train.log](https://github.com/SPengLiang/DID-M3D/files/9230190/train.log)
and your model by used this code to get 25.38(used 4 gpu，batch size 16/gpu)?

**Comment by SPengLiang** [comment]
Yeah. Thank you for providing the log. I will give you feedback soon.

**Comment by SPengLiang** [comment]
Due to the GPU memory limitation, I re-trained the model on a single GPU with batchsize 8 (OOM when batchsize is 16).
I obtain the results below at 100 epoch:
<img width="1351" alt="image" src="https://user-images.githubusercontent.com/14855845/182079473-2a5288a2-4dd9-489e-a8e4-d5d74b1c2888.png">

log here:
[train.log](https://github.com/SPengLiang/DID-M3D/files/9231216/train.log)

BTW, in your log, I find that depth loss is 9.6392, 9.6733, 9.6814, 9.6511, 9.4406 in epoch [8,12], which seems that the depth loss didn't decline. I cannot reproduce your results. So if you want to use a single GPU to train, you can try batchsize 8.

**Comment by MillX2021** [comment]
thanks for you. and I seems the depth loss didn't decline too. I try to use 4gpu train, if i get the result , i will upload log file.
---------------Update-----------------------
I get AP 24.54 when i used 4gpu。
![image](https://user-images.githubusercontent.com/11409446/182112050-a6a6d01e-4246-42d9-a21f-2ff33095c601.png)

**Comment by SPengLiang** [comment]
Thanks for your feedback. As aforementioned, you can try batchsize 8 when you use only one GPU. If you use batchsize 16 on a single GPU, you may need to carefully tune the learning rate. Good luck!
